[PD-3274] improve criteria presentation scroll

mwarnsleyDE opened the pull request:
## What's this PR do?
PR updates the main sidebar navigation on desktop and mobile with the breadcrumbs and navigation

## Screenshots (if appropriate)
![screen shot 2017-06-27 at 3 57 57 pm](https://user-images.githubusercontent.com/22029077/28367280-4391e23a-6c5e-11e7-9256-92d622853858.png)
![screen shot 2017-06-27 at 3 58 24 pm](https://user-images.githubusercontent.com/22029077/28367286-48eeefac-6c5e-11e7-9935-cf02399e9cba.png)


## Where should the reviewer start?
Start inside any of the analytics applications

## How should this be manually tested?
Pull down the repository. Go inside of analytics and look at the sidebar in desktop and mobile and you will see the changes. The screenshots are updated in this PR as well to show what they should look like. Along with that, if you go inside of the myjobs folder and run dkgg watch you will see the test run from editing the main reducer test to include the mobilecrumbs for the sidebar

## Any background context you want to provide?
If you find any commented out code or anything that needs updated please let me know. There were a decent amount of files changed and stuff flipped around so I lost track a bit and went back through but I could have missed something.

aljordan82 commented on gulp/src/sass/analytics/_analytics_media_queries.scss:
```diff
@@ -194,14 +53,36 @@
+         z-index: 9;
+       }
+       .tab {
+         @include transition(all, 500ms, ease-out, 0s);
```
I would remove transition on .tab because it shows a triangle.

aljordan82 commented:
I think it would be better if we keep the Search Company styles as they used to be (coloring changes are fine, but changing sizes make the dropdown look broken
Also, the selected company font color needs a higher contrast level
)![image](https://user-images.githubusercontent.com/6749055/28371472-11473278-6c6b-11e7-878f-c6fe904c6370.png)
![image](https://user-images.githubusercontent.com/6749055/28371502-280dc53a-6c6b-11e7-9280-8612ff8d288f.png)

strempe commented:
<img width="349" alt="screen shot 2017-07-19 at 11 08 56 am" src="https://user-images.githubusercontent.com/25237191/28374658-701027e4-6c73-11e7-8641-e80c6f64d71b.png">

Is the date a value that is needed for Custom Dimensions>First Viewed On?


<img width="345" alt="screen shot 2017-07-19 at 11 09 34 am" src="https://user-images.githubusercontent.com/25237191/28374733-9d365004-6c73-11e7-8ab6-a7ec755c47f7.png">

The title needs to be more specific (I don't think this was focused on enough in the mockups)
ie: Job View by State: Flordia - If 'Flordia' could be moved to the gray nav where the calendar lives, we won't have to worry about making it fixed.

strempe commented:
<img width="362" alt="screen shot 2017-07-19 at 11 08 02 am" src="https://user-images.githubusercontent.com/25237191/28374990-52f3de84-6c74-11e7-938a-97f2b2866313.png">

Positioning: 
Tabs calendar and date. 
The date is cropped off the page.

mwarnsleyDE commented:
@strempe @aljordan82 
Navigation: Fixed
Company Select: Restyled and fixed for mobile and desktop
Tooltip: We've discussed that before and it's been an issue for awhile. Also there aren't any tooltips on Mobile
X Close Button: @strempe not seeing this on my machine or on Aladeens screenshots either
Data Value: That comes with the time that it was viewed on for that
Naming Convention: Comes from the backend guys and whatever they send me
Overlapping Tabs: This has been an issue for awhile too. This has to be thought of better and implemented in a different/better way because it's hard when you can have so many tabs along with having a calendar and other stuff to the right when you start getting into smaller screens

If you guys see anything else or anything please let me know. Trying to get this merged as soon as possible. Make sure to do a pull from the branch when you are back on it and testing as there are many changes that have been done that need to be pulled into your branches.

mwarnsleyDE commented:
@aljordan82 could you possibly take a look at this and see if I messed something up with the table Pagination. I haven't touched it and for some reason if you have a ton of pages to go through you can't go any farther than the second page. I have been looking but my eyes are burnt out right now and I can't seem to track where I messed up with this at LOL. Just need a second pair of eyes.

aljordan82 commented:
I m seeing the tooltip/popover positioned incorrectly at 950px width
